dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/LogThreadStatistics.java

```diff
@@ -38,7 +39,7 @@ public void run(Program program) {
                     storeCount++;
                 } else if (e instanceof Load) {
                     loadCount++;
-                } else if (e instanceof Fence) {
+                } else if (e.hasTag(Tag.FENCE)) {
```

Comment:
Maybe we should change this to `e instance GenericVisibleEvent`, cause the intention of the program statistics is to give a rough understanding of all the visible events to gauge the size of the execution graphs.
Like this, you will miss visible non-fences (like the ones in Vulkan).